core/test/utils/batch_helpers.hpp

```diff
@@ -106,11 +106,11 @@ std::unique_ptr<MatrixType> generate_3pt_stencil_batch_matrix(
         {}};
     for (int row = 0; row < num_rows; ++row) {
         if (row > 0) {
-            data.nonzeros.emplace_back(row - 1, row, value_type{-1.0});
+            data.nonzeros.emplace_back(row - 1, row, value_type{-0.5});
```

Comment:
did the previous value lead to any issues?

Reply:
Just wanted to test for a more positive matrix. Cg sometimes seems to have issues.